### PR TITLE
fix pep8 violations

### PR DESCRIPTION
**Pull request by joren485**
This PR fixes all pep8 violations:
* trailing whitespace
* line too long
* do not assign a lambda expression, use a def

And a typo of "sandwich" is fixed.

### CODE REVIEW

**Comment by thomwiggers** [comment]
Hey, I had dibs on typos

**Comment by pingiun** [comment]
Hey I had dibs on slackbot

File: bot.py [hunk 1]
```diff
@@ -29,7 +29,10 @@
 
 
 class Bot:
-    """A minimal wrapper for the Slack API, this class also manages the database"""
+    """
+    A minimal wrapper for the Slack API,
+    this class also manages the database
+    """
```
Comment (pingiun) [inline bot.py:35]: Why break this up if it fits on one line?

File: bot.py [hunk 2]
```diff
@@ -167,14 +186,20 @@ def create_wbw_session():
 
 
 def wbw_get_lowest_member(voted):
-    """Looks up the wiebetaaltwat balance and returns the slack name of the lowest standing balance holder
+    """
```
Comment (pingiun) [inline bot.py:189]: The first line of a docstring should start on the same line as the opening """

File: bot.py [hunk 3]
```diff
@@ -213,37 +251,68 @@ def check(bot):
...
-        # The multiple max allows us to make a random choice when the vote is tied
+
+        # The multiple max allows us to make a random choice when the vote
+        # is tied
         try:
-            choice = random.choice(multiple_max(votes, key=lambda x: x[1]))[0]
+
+            highest_vote = max(votes, key=lambda i: i[1])[1]
+
+            choice = random.choice(
```
Comment (pingiun) [inline bot.py:280]: You should add a comment here because you replaced a fully explained function with a one-liner

**Comment by pingiun** [comment]
Hey @thomwiggers you merged this PR very quickly even though it isn't a trivial change

File: bot.py (hunk 2 above)
Comment (joren485) [reply bot.py]: From pep8:
> For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.

The line is 73 chars (including the space after ",").

File: bot.py (hunk 1 above)
Comment (joren485) [reply bot.py:35]: From pep8:

>For flowing long blocks of text with fewer structural restrictions (docstrings or comments), the line length should be limited to 72 characters.

Comment (pingiun) [reply bot.py:35]: Nvm my pycharm shows a ruler at the wrong column

**Pull request merged**